Chaincase.Common/Chaincase.Common.csproj

```diff
@@ -11,6 +11,7 @@
       <PackageReference Include="ReactiveUI" Version="13.2.2" />
       <PackageReference Include="ReactiveUI.Blazor" Version="13.2.2" />
       <PackageReference Include="NBitcoin" Version="5.0.76" />
+      <PackageReference Include="BIP78.Receiver" Version="0.2.0" />
```

Comment:
Did not end up using. Delete this

Reply:
[https://www.google.com/search?q=delet+this+teletubby&client=firefox-b-1-d&tbm=isch&source=iu&ictx=1&fir=_Nvzraa5NyGf7M%252CZJ_42XsJP5rZ1M%252C_&vet=1&usg=AI4_-kTTbUyDC4hwXSHQyYuPn7S4DwjRUg&sa=X&ved=2ahUKEwiYp7Xx7OLvAhXTUjUKHaSKD98Q9QF6BAgaEAE#imgrc=_Nvzraa5NyGf7M]